pkg/podinformer/podinformer.go

```diff
@@ -212,21 +213,29 @@ func (p *PodInformer) syncToStdout(key string) error {
 			if s.ContainerID == "" {
 				continue
 			}
+
+			var containerID string
+			containerIDSplit := strings.Split(s.ContainerID, "://")
+			if len(containerIDSplit) == 1 {
+				containerID = containerIDSplit[0]
+			} else if len(containerIDSplit) == 2 {
+				containerID = containerIDSplit[1]
+			}
```

Comment:
If `len(containerIDSplit) > 2`, we could `continue` after printing an error with `log.WithFields(...).Error("unrecognised containerID")` instead of using the empty string.

Reply:
Updated.